ci-operator/jobs/syndesisio/syndesis-qe/syndesisio-syndesis-qe-1.15.x-periodics.yaml

```diff
@@ -0,0 +1,76 @@
+periodics:
+- agent: kubernetes
+  cluster: build05
+  cron: 0 1 * * 1
+  decorate: true
+  decoration_config:
+    skip_cloning: true
+  extra_refs:
+  - base_ref: 1.15.x
+    org: syndesisio
+    repo: syndesis-qe
+  labels:
+    ci-operator.openshift.io/cloud: aws
+    ci-operator.openshift.io/cloud-cluster-profile: aws-cspi-qe
+    ci-operator.openshift.io/variant: fuse-online-ocp4.13-lp-interop
+    ci.openshift.io/generator: prowgen
+    job-release: "4.13"
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-syndesisio-syndesis-qe-1.15.x-fuse-online-ocp4.13-lp-interop-fuse-online-interop-aws
```

Comment:
You may need to add the [Slack alerts](https://cspi-qe.gitbook.io/ocp-ci-onboarding/tutorials/reporting_guide#how-to-setup-slack-alerts-for-a-scenario) to PQE Slack channel & make sure its public

Reply:
Just [opened a ticket](https://issues.redhat.com/browse/LPTOCPCI-254) to address this. Thanks!

Reply:
Here is also an example: https://github.com/openshift/release/pull/38738/files#diff-47bf248bb2bc930bf70d6c658fea67b33d284b93037c717d4dd254bdd9fe3e9aR22